ios/MullvadVPN/TunnelManager/Tunnel.swift

```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import NetworkExtension
+import UIKit
```

Comment:
FYI: Why do we need UIKit import in Tunnel?

Reply:
Good point. We don't need it there. Removed it.